src/cgame/cg_event.cpp

```diff
@@ -1098,7 +1098,7 @@ void CG_EntityEvent( centity_t *cent, vec3_t position )
 
 				if ( !CG_IsTrailSystemValid( &source->muzzleTS ) )
 				{
-					source->muzzleTS = CG_SpawnNewTrailSystem( cgs.media.teslaZapTS );
+					source->muzzleTS = CG_SpawnNewTrailSystem( cgs.media.reactorZapTS );
```

Comment:
Also rename the EV_TESLATRAIL above?

Reply:
this can be done in another PR, the whole purpose of this PR is to sync with asset repositories.